exercises/diffie-hellman/example.js

```diff
@@ -19,14 +19,14 @@ var DiffieHellman = function(p, g){
     return Math.pow(theirPublicKey, ourPrivateKey) % this.p;
   };
 
-  this.validateInitialArguments = function (p, g){
+  this.validateInitialArguments = function (this.p, this.g) {
```

Comment:
@rchavarria @matthewmorgan Let me know if this change is what you had in mind.

Reply:
Nevermind.  Tests are failing with that change.

Reply:
So maybe `prime modulus` instead of this.p and `generator` instead of this.g?  I don't want to do anything convoluted to resolve this issue.